src/pages/HomePage.js

```diff
@@ -6,15 +6,17 @@ export default function HomePage({
   notes,
   onDeleteNote,
   handleCardToggle,
+  _id,
 }) {
   return (
     <>
       <Header />
       <PastTripList
         onHandleNewNote={onHandleNewNote}
         notes={notes}
-        onDeleteNote={onDeleteNote}
+        onDelete={onDeleteNote}
```

Comment:
maybe you can rename onDelete to onDeleteNote in you entire code? Just do be more clean. 
But this is also preference :)